sycl/test/acap/test_aie_cstreams.cpp

```diff
@@ -0,0 +1,69 @@
+// REQUIRES: acap
+
+// RUN: %acap_clang %s -o %s.bin | FileCheck %s -check-prefix CHECK-MERGING
+// RUN: %add_acap_result %s.bin
+// RUN: rm %s.bin
+
+// check that we only compile 3 device kernel via chess
+// CHECK-MERGING-COUNT-3: Linking Kernel
+// CHECK-MERGING-NOT: Linking Kernel
+
+#define TISYCL_DEVICE_ALLOCATOR_DEBUG
```

Comment:
What is this?
`TISYCL` instead of `TRISYCL`?
It looks like this is copy-pasted everywhere.